### PR TITLE
Update multi-cpu-architectures.md

### PR DESCRIPTION
**Pull request by Lukazo0**
Updated all relevant places, except the matrix part, which already shows how the matrix works and adding another paragraph for arm64-graviton2 would have made it more confusing (I can still add that if necessary, tho 😊 )

NOTE: Hugepages section updates have been collected from internet research

### CODE REVIEW

**Review by nrios14 — DISMISSED** [review]

**Review by michal-at-travisci — APPROVED** [review]

**Review by michal-at-travisci — APPROVED** [review]
Made necessary changes

File: user/multi-cpu-architectures.md [hunk 1]
```diff
@@ -35,28 +57,29 @@ arch:
...
 ```
 {: data-file=".travis.yml"}
 
 If you are already using a [build matrix](/user/customizing-the-build/#build-matrix) to test multiple versions, the `arch` key also multiplies the matrix.
 
-- The ppc64le (IBM Power) and s390x (IBM Z) build jobs are run in an LXD compliant Linux OS image. 
-- The arm64 CPU architecture build job is run in an LXD compliant Linux OS image.
+- The `ppc64le` (IBM Power) and `s390x` (IBM Z) build jobs are run in an LXD compliant Linux OS image. 
+- The `arm64` CPU architecture build job is run in an LXD compliant Linux OS image.
+- The `arm64-graviton2` architecture builds can be run on both LXD and regular 'full VM' environments. You **need** explicitely set target environment by using `virt` key. A `virt: vm` routes build jobs to full virtual machine setup while `virt: lxd` routes build jobs to LXD container setup.
```
Comment (nrios14) [inline user/multi-cpu-architectures.md:69]: You need **to** explicitly set **the** target environment by using the 'virt' key. A `virt: vm` routes build jobs to **a** full virtual machine setup while `virt: lxd` routes build jobs to **an** LXD container setup.

File: user/multi-cpu-architectures.md [hunk 2]
```diff
@@ -147,31 +174,7 @@ You can try it out also for `ppc64le` (IBM Power) and `s390x` (IBM Z) based dock
...
-
-> System doesn't support syscall interception
-
-It most probably means a system call interception is outside of the list of the ones considered to be safe (LXD can allow system call interception [if it's considered to be safe](https://github.com/lxc/lxd/blob/master/doc/syscall-interception.md)). 
 
-## Hugepages Support from within LXD Container
+## LXD related limitations
 
-A build job can’t enable hugepages within an LXD container - this is something that may change in the future, yet it depends on potential Linux kernel changes, which is something that needs to be reviewed and developed.
+Pleae have a look at [Build Environment Overview](/user/reference/overview/#linux-security-and-lxd-container) for more details.
```
Comment (nrios14) [inline user/multi-cpu-architectures.md:180]: For more details, see [Build Environment Overview](/user/reference/overview/#linux-security-and-lxd-container)

Also, there is a typo on the Please it currently reads "Pleae"

**Review by nrios14 — COMMENTED** [review]
made some minor changes and fixed a typo

File: user/multi-cpu-architectures.md (hunk 1 above)
Comment (michal-at-travisci) [reply user/multi-cpu-architectures.md:69]: Updating on master, thanks!

File: user/multi-cpu-architectures.md (hunk 2 above)
Comment (michal-at-travisci) [reply user/multi-cpu-architectures.md:180]: Fixing on master, thanks!

**Pull request merged**